src/test/java/dk/sdu/se_f22/searchmodule/misspellings/MisspellingsTest.java

```diff
@@ -0,0 +1,139 @@
+package dk.sdu.se_f22.searchmodule.misspellings;
+
+import dk.sdu.se_f22.sharedlibrary.db.DBConnection;
+import dk.sdu.se_f22.sharedlibrary.db.DBMigration;
+import org.junit.jupiter.api.*;
+
+import java.sql.*;
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class MisspellingsTest {
+
+    Misspellings misspelling = new Misspellings();
+    ArrayList<String> listWrong = new ArrayList<>();
+    ArrayList<String> listCorrect = new ArrayList<>();
+
+    @BeforeAll
+    static void dbMigration(){
+        DBMigration migrator = new DBMigration();
+        migrator.migrate();
```

Comment:
I just need to be sure, what it is you want here?
Is it that you want to empty the database?
Then you need to use DBMigrationFresh class - This will probably do nothing, unless the only thing is to ensure everything is up to date.